messenger/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .settings import *
```

Comment:
Oh. That was totally unclear to me. I'm used to running something with the
word make, so the new way was unintuitive for me. Is there a way make.py
can still run?

I don't think the installation will ever be simple for mex files to "just
work" with a setup.py... I think the easiest thing will actually be to
provide mex files statically linked to libzmq that can just be run by
MATLAB. My $.02 at least.

On Wed, Apr 8, 2015 at 10:03 PM, Sang Han <notifications@github.com> wrote:

> In messenger/__init__.py
> <https://github.com/arokem/python-matlab-bridge/pull/173#discussion_r28035471>
> :
>
> > @@ -0,0 +1,2 @@
> > +from .settings import *
>
> The purpose of turning messenger info a module was whaf I felt a natural
> evolution of the code base. I wanted go separate files to help manage
> dependencies, keep code DRY and hook into the setuptools tool chain to
> manage complication
>
> Currently the entire module can be run as a script. Whereas in the past
> you would run
>
> cd messenger
> python make.py matlab
> cd ..
>
> You would now just run
>
> python -m messenger matlab
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/arokem/python-matlab-bridge/pull/173/files#r28035471>.
>